### DISCUSSION TITLE
Автоподбор параметров

### DISCUSSION THREAD

**Discussion by hikeri** [hufrea/byedpi#68] [Ideas]
Будет ли добавлена фунция автоподбора параметров прямо как в [dpi tunnel](https://github.com/nomoresat/DPITunnel-cli)?

**Comment by hufrea** [comment]
В сам бинарник нет, но, возможно, будет добавлен shell-скрипт для подбора параметров. Сразу стоит сказать, что полученные параметры вряд-ли будут  хорошо работать, т.к. разные сайты могут блокировать по различным правилам, и вручную подбирать оптимальные настройки все же придется.
